erpnext/crm/doctype/lead/lead.py

```diff
@@ -24,21 +23,36 @@ class Lead(SellingController, CRMNote):
 	from typing import TYPE_CHECKING
 
 	if TYPE_CHECKING:
-		from frappe.types import DF
-
 		from erpnext.crm.doctype.crm_note.crm_note import CRMNote
+		from erpnext.crm.doctype.lead_channel.lead_channel import LeadChannel
+		from erpnext.crm.doctype.lead_diamond.lead_diamond import LeadDiamond
+		from erpnext.crm.doctype.lead_product_item.lead_product_item import LeadProductItem
+		from frappe.types import DF
 
+		account_number: DF.Data | None
+		address: DF.Data | None
 		annual_revenue: DF.Currency
+		bank_branch: DF.Literal[None]
+		bank_district: DF.Literal[None]
+		bank_name: DF.Literal["Agribank", "ACB", "BIDV", "Vietcombank", "Vietinbank", "Techcombank", "Sacombank", "MB Bank", "Eximbank", "VPBank", "SHB", "Nam A Bank", "BAC A BANK", "OceanBank", "TPBank"]
```

Comment:
issue: The others are using capital, please keep it consistent
```suggestion
		bank_name: DF.Literal["Agribank", "ACB", "BIDV", "Vietcombank", "Vietinbank", "Techcombank", "Sacombank", "MB Bank", "Eximbank", "VPBank", "SHB", "Nam A Bank", "Bac A Bank", "OceanBank", "TPBank"]
```